randovania/games/cave_story/gui/dialog/game_export_dialog.py

```diff
@@ -34,6 +36,14 @@ def __init__(self, options: Options, patch_data: dict, word_hash: str, spoiler:
 
         # Output
         self.output_file_button.clicked.connect(self._on_output_file_button)
+        self.tweaked_radio.clicked.connect(self._on_platform_radio_clicked)
+        self.freeware_radio.clicked.connect(self._on_platform_radio_clicked)
+
+        # Target Platform
+        if per_game.platform is CSPlatform.FREEWARE:
+            self.freeware_radio.setChecked(True)
+        else:
+            self.tweaked_radio.setChecked(True)
```

Comment:
does this set the correct text when closing/reopening the dialog?

Reply:
doesn't actually, but i'm not sure how to fix that. There should be an event to check for change in the radio stuff rather than using the click event?

Reply:
just add a function that updates the text based on the current state, and call it after both this block and in the click event